db/migrate/20171129121738_change_entity_type.rb

```diff
@@ -0,0 +1,6 @@
+class ChangeEntityType < ActiveRecord::Migration
+  def change
+    change_column :organizations, :entity_type, 'integer USING CAST(entity_type AS integer)'
```

Comment:
I don't understand why this `integer USING CAST(entity_type AS integer)` is neccesary neither the "be careful with the migration" comment on the PR description or other texts helps to get it... Are you trying to do an enumerator?

Reply:
entity_type its defined as enumerator in the model but not changed in the data base,
so I make this migration.

Reply:
Okey, you're trying to convert existing values 👍  nice! Something that could go wrong about this? I mean the warning means that you haven't tested that this converts always the existing values to corresponding integers? If so give more context for the maintainers of the preproduction app to doublecheck

Reply:
no, there should not be any value in that field and if they are, they will be established automatically. I only call attention because I find an atypical migration. should clarify the text of thePR